articles/migrations/guides/calling-api-with-idtokens.md

```diff
@@ -7,17 +7,21 @@ topics:
   - management-api
   - id-tokens
   - tokens
+contentType: 
+    - concept
+    - how-to
+    - reference
 ---
 
 # Migration Guide: Management API and ID Tokens
 
 For some use cases you could use [ID Tokens](/tokens/id-token) in order to call some of the [Users](/api/management/v2#!/Users/get_users_by_id) and [Device Credentials](/api/management/v2#!/Device_Credentials/get_device_credentials) endpoints of the Management API. 
 
-This functionality is being deprecated. You will have to use proper [Access Tokens](/tokens/access-token) in order to access any of the endpoints of the [Management API](/api/management/v2).
+The capabilities for using ID Tokens to authorize some of the Users and Device Credentials endpoints of the Management API are being deprecated. You will have to use proper [Access Tokens](/tokens/access-token) in order to access any of the endpoints of the [Management API](/api/management/v2). Make sure the `Allow ID Tokens for Management API v2 Authentication` toggle is turned off after completing the migration to Access Tokens.
```

Comment:
@rkhoriander Why we changed this sentence? This way we repeat what the exactly previous sentence says (in a more complex way).

Reply:
+ the new wording is also inaccurate/unclear

Reply:
@mpaktiti @rkhoriander I changed the sentence to one that Sole wanted. I agree it seems overly complex. @jeffreylees Is there a better, more accurate way of saying it?

Reply:
I would either do one of two things

1. Leave it as is. It says what it needs to say.

2. If for some reason it must be changed, remove the line previous to it too (so that we aren't just repeating ourselves) and say something like: 

> The use of ID Tokens to authorize requests to the Users and Device Credentials endpoints of the Management API is a deprecated behavior. You will...

Reply:
From what we understood, these were language changes approved by Hilary Elgin and requested by Sole. That said, I think the whole point was to add the last sentence (that the toggle needs to be turned off after completing the migration). So I went back to the old, clearer language and just left that last sentence. Take a look.